gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,655 @@
+                "max": "auto",
+                "min": "auto",
+                "scale": "linear"
+            }
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 13,
+            "isShared": false,
+            "legend": false,
+            "markers": [],
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.http_request_duration_seconds.count{*}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Http request duration",
+            "type": "timeseries",
+            "viz": "timeseries",
+            "width": 43,
+            "x": 34,
+            "y": 29,
+            "yaxis": {
+                "includeZero": true,
+                "label": "",
+                "max": "auto",
+                "min": "auto",
+                "scale": "linear"
+            }
+        },
+        {
+            "add_timeframe": true,
+            "auto_refresh": false,
+            "background_color": "gray",
+            "bgcolor": "gray",
+            "content": "Requests Information",
+            "font_size": "18",
+            "height": 10,
+            "html": "Requests Information",
+            "isShared": false,
+            "refresh_every": 30000,
+            "scaleFactor": 1,
+            "show_tick": true,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 43,
+            "x": 34,
+            "y": 1
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 9,
+            "isShared": false,
+            "precision": 1,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "precision": 1,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "conditional_formats": [],
+                        "q": "avg:gitlab.pipelines_created_total{*}.as_count()"
```

Comment:
This is a monotonic count, so the widget will only display the number of new pipelines since the last check run and not the number of pipelines.
This metric seems a good candidate to be sent as a gauge/count instead of a monotonic count. WDYT?